kanidm_book/src/pam_and_nsswitch.md

```diff
@@ -124,6 +131,11 @@ and be authorised to start interactive sessions. This is configured through a st
 that are executed in order to evaluate the request. This is done through a series of steps
 where each module may request or reuse authentication token information.
 
+When a service like sudo, sshd, su etc. wants to authenticate someone, it opens the pam.d config of that service, 
```

Comment:
Something about this text feels awkward where it is in this section, but I do want an explanation about how pam works. Could we possible make a breakout chapter in the faq about "how pam/nsswitch work in detail" instead? And then leave this section as "how to configure kanidm for unix integration" specifically.

Reply:
Yeah that would be good. I can write something there.